lnrpc/routerrpc/router_server.go

```diff
@@ -1176,7 +1178,8 @@ func toPairSnapshot(pairResult *PairHistory) (*routing.MissionControlPairSnapsho
 // getPair validates the values provided for a mission control result and
 // returns the msat amount and timestamp for it.
 func getPair(amtMsat lnwire.MilliSatoshi, amtSat btcutil.Amount,
-	timestamp int64) (lnwire.MilliSatoshi, time.Time, error) {
+	timestamp int64, isFailure bool) (lnwire.MilliSatoshi, time.Time,
```

Comment:
I think based on the usage this actually means `isSuccess`?

Reply:
that was a mistake in my thinking indeed, updated.